include/cantera/kinetics/ElectronCollisionPlasmaRate.h

```diff
@@ -0,0 +1,153 @@
+//! @file ElectronCollisionPlasmaRate.h   Header for plasma reaction rates parameterized
+//!     by electron collision cross section and electron energy distribution.
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_ELECTRONCOLLISIONPLASMARATE_H
+#define CT_ELECTRONCOLLISIONPLASMARATE_H
+
+#include "cantera/thermo/PlasmaPhase.h"
+#include "cantera/kinetics/ReactionData.h"
+#include "ReactionRate.h"
+#include "MultiRate.h"
+
+namespace Cantera
+{
+
+//! Data container holding shared data specific to ElectronCollisionPlasmaRate
+/**
+ * The data container `ElectronCollisionPlasmaData` holds precalculated data common to
+ * all `ElectronCollisionPlasmaRate` objects.
+ */
+struct ElectronCollisionPlasmaData : public ReactionData
+{
+    ElectronCollisionPlasmaData();
+
+    virtual bool update(const ThermoPhase& phase, const Kinetics& kin) override;
+    using ReactionData::update;
+
+    virtual void invalidateCache() override {
+        ReactionData::invalidateCache();
+        EnergyLevels.resize(0);
+        Distribution.resize(0);
+    }
+
+    Eigen::ArrayXd EnergyLevels; //!< electron energy levels
+    Eigen::ArrayXd Distribution; //!< electron energy distribution
+};
+
+
+//! Electron collision plasma reaction rate type
+/*!
+ * The electron collision plasma reaction rate uses the electron collision
+ * data and the electron energy distribution to calculate the reaction rate.
+ *
+ *   \f[
+ *        k_f =  \gamma \int_0^{\infty} \epsilon \sigma F_0 d\epsilon
+ *   \f]
```

Comment:
```suggestion
 *   @f[
 *        k_f =  \gamma \int_0^{\infty} \epsilon \sigma F_0 d\epsilon
 *   @f]
```
Based on recent updates, doxygen should use `@` rather than `\`.